### ISSUE TITLE
Password isn't properly escaped

### ISSUE THREAD

**Issue by gadgetvirtuoso** [hjdhjd/homebridge-myq#942]
**Describe The Problem:**
If the password contains some special characters, the plugin isn't able to authenticate correctly

**To Reproduce:**
Set password with special characters. I know for sure "!" will break the password field.

Homebridge will then show: "myQ API: Invalid myQ credentials given. Check your login and password."

**Logs:**

```
Show the Homebridge logs here.
Remove any sensitive information.
```

**Homebridge Configuration:**

```json
Show the relevant portion of your homebridge config.json here, if needed.
Remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **Homebridge Version:** 
* **Node Version:** 
* **Homebridge-myQ Plugin Version**:
* **Apple Device and iOS / macOS / iPadOS / tvOS Version:**
* **Operating System and OS Version:**

**Comment by Raiden38** [comment]
I can confirm the issue above. I had an "!" in my password and it was not working anymore. Changed my password and everything is back to normal. Thanks!

**Comment by hjdhjd** [comment]
Thanks - I’ll look into this one. Appreciated.

**Comment by dapostman** [comment]
I get the same error. Thought it might be due my using strong passwords from Apple, using a "-". Switched to "No Special Characters". Same error.

Log line: [12/24/2022, 5:18:10 PM] Homebridge v1.6.0 (HAP v0.11.0) (Homebridge Hades Canyon) is running on port 51719.
[12/24/2022, 5:18:11 PM] [myQStuff] myQ API: Invalid myQ credentials given. Check your login and password.

Environment:

Homebridge Version: 1.60
Node Version: v16.15.0
Homebridge-myQ Plugin Version: homebridge-myq v2.8.1
Apple Device and iOS / macOS / iPadOS / tvOS Version: 3.1Ghz Quad-core Intel Core i7 Mac Mini
Operating System and OS Version: 13.0 (Ventura)

**Comment by hjdhjd** [comment]
Locking this issue - appreciate the confirmations, just don’t need anymore data at this point. Workarounds are: don’t use those special characters in your password if you have an issue. Will be addressed in a future update.

**Comment by hjdhjd** [comment]
Addressed in v2.8.2. Enjoy!
